sentry/sentry.py

```diff
@@ -46,22 +54,57 @@ def cog_unload(self):
         self.bot.remove_before_invoke_hook(self.before_invoke)
         return super().cog_unload()
 
+    @checks.mod()
+    @commands.group(name="sentry", pass_context=True)
+    async def _sentry(self, ctx: commands.context.Context):
+        """Automatically reply to messages matching certain trigger phrases"""
```

Comment:
Docstring should be related to Sentry cog

Reply:
Might've just copied that doc string from the examples 😂